display/3.interval_instances.yaml

```diff
@@ -0,0 +1,13 @@
+:type: map
+:entries:
+  - :name: lttng:host
+    :samples:
+      - :values: ['"iris01"',1289,1098,1,0,'"getDeviceId"',2,0]
+        :times: 5
+      - :values: ['"iris01"',1289,1098,1,0,'"getDeviceName"',2,0]
+        :times: 5
+      - :values: ['"iris02"',1289,1098,1,0,'"getDeviceId"',2,0]
+        :times: 5
+      - :values: ['"iris02s"',1289,1098,1,0,'"getDeviceName"',2,0]
+        :times: 5   
+ 
```

Comment:
Please also add tests for all types of interval messages.

Reply:
Done